lib/config-generator.js

```diff
@@ -127,6 +127,16 @@ export class ConfigGenerator {
         if (this.answers.languages.includes("md")) {
             Object.assign(this.answers, await enquirer.prompt(mdQuestions));
         }
+
+        if (this.answers.configFileLanguage === "ts") {
+            const nodeVersion = process.versions.node.split(".").map(Number);
+
+            // Node.js v24.3.0 removed the experimental warning from type stripping.
+            if (nodeVersion[0] < 24 || (nodeVersion[0] === 24 && nodeVersion[1] < 3)) {
```

Comment:
[Refactor] [`semver`](https://www.npmjs.com/package/semver) is already a dependency. I think it'd be cleaner & safer to use `semver.lt` here.